src/Symfony/Component/Form/FormRendererEngineInterface.php

```diff
@@ -21,11 +21,13 @@
     /**
      * Sets the theme(s) to be used for rendering a view and its children.
      *
-     * @param FormView $view   The view to assign the theme(s) to
-     * @param mixed    $themes The theme(s). The type of these themes
-     *                         is open to the implementation.
+     * @param FormView $view             The view to assign the theme(s) to
+     * @param mixed    $themes           The theme(s). The type of these themes
+     *                                   is open to the implementation.
+     * @param bool     $useDefaultThemes if set to false, will not use
+     *                                   default themes to render the view
      */
-    public function setTheme(FormView $view, $themes);
+    public function setTheme(FormView $view, $themes, $useDefaultThemes = true);
```

Comment:
adding the argument in the interface is a BC break. So this cannot be done before 4.0 (and 3.4 should trigger a deprecation warning in case it finds an implementation of the interface without the extra optional attribute)

Reply:
Okay, makes sense. How should I proceed then?